### PR TITLE
Shellcheck

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

File: mender-convert [hunk 1]
```diff
@@ -56,33 +56,33 @@ trap trap_exit EXIT
...
+  echo "You must execute mender-convert from the root directory: ${mender_convert_dir}"
+  exit 1
 fi
 
 if [ -z "${MENDER_ARTIFACT_NAME}" ]; then
   echo "Sorry, it seems that you have not defined MENDER_ARTIFACT_NAME"
   echo "You can do this with the following command:"
   echo ""
-  echo -e "\tMENDER_ARTIFACT_NAME="release-1" ./mender-convert"
+  echo -e "\tMENDER_ARTIFACT_NAME=\"release-1\" ./mender-convert"
```
Comment (lluiscampos) [inline mender-convert]: Review your commits wrt this line: your first commit just removes the quotes, the indentation commit adds the `\`

Comment (oleorhagen) [reply mender-convert]: Nice, thx

File: mender-convert [hunk 2]
```diff
@@ -82,7 +82,7 @@ if [ -z "${MENDER_ARTIFACT_NAME}" ]; then
...
```
Comment (mirzak) [inline mender-convert]: Now the commit message should be updated to `Tidying: Escape extra double quotes` :)

Comment (oleorhagen) [reply mender-convert]: haha, damn!

**Comment by oleorhagen** [comment]
Also went through all the other shell files present

File: scripts/run-tests.sh [hunk 3]
```diff
@@ -136,18 +136,18 @@ get_pytest_files
 test_result=0
 
 convert_and_test "qemux86_64" \
-                 "release-1" \
-                 "${UBUNTU_IMAGE_URL}" \
-                 "${UBUNTU_IMAGE}" \
-                 "${UBUNTU_IMAGE}.gz" \
-                 "configs/qemux86-64_config" || test_result=$?
+  "release-1" \
```
Comment (oleorhagen) [inline scripts/run-tests.sh]: I don't really know if these should follow 2-space indent or not?

File: mender-convert-package [hunk 4]
```diff
@@ -95,11 +95,11 @@ alignment_sectors=$((${MENDER_PARTITION_ALIGNMENT} / 512))
 
 if [ "${MENDER_GRUB_EFI_INTEGRATION}" == "y" ]; then
   boot_part_start=${alignment_sectors}
-  overhead_sectors=$(( ${alignment_sectors} * 4 ))
```
Comment (mirzak) [inline mender-convert-package]: You have a lot of these changes where you are removing "whitespaces" and would try no to mix this with indentation fixes.

Also in this particular case I vote to have extra white space before/after for `$((  ))` statements for readability purposes. This probably applies to many other locations and would simple exude these changes for now

Comment (oleorhagen) [reply mender-convert-package]: Alright, fixing up

File: scripts/run-tests.sh (hunk 3 above)
Comment (mirzak) [reply scripts/run-tests.sh]: Might as well

File: modules/disk.sh [hunk 5]
```diff
@@ -65,17 +65,16 @@ disk_extract_part() {
 #
 #  $1 - MiB value
 disk_mb_to_sectors() {
-    echo "$(( (${1} * 1024 * 1024) / 512 ))"
+  echo "$(((${1} * 1024 * 1024) / 512))"
```
Comment (mirzak) [inline modules/disk.sh]: You removed whitespaces here :)

File: modules/disk.sh [hunk 6]
```diff
@@ -65,17 +65,16 @@ disk_extract_part() {
...
 }
 
 # Convert 512 sectors to MiB
 #
 #  $1 - number of 512 sectors
 disk_sectors_to_mb() {
-    echo "$(( (${1} * 512) / 1024 / 1024 ))"
+  echo "$(((${1} * 512) / 1024 / 1024))"
```
Comment (mirzak) [inline modules/disk.sh]: You removed whitespaces here :)

**Pull request merged**